fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -692,7 +693,7 @@ ACTOR Future<Void> sendGrvReplies(Future<GetReadVersionReply> replyFuture,
 
 	double end = g_network->timer();
 	for (GetReadVersionRequest const& request : requests) {
-		double duration = end - request.requestTime();
+		double duration = end - request.requestTime() - request.proxyTagThrottledDuration;
```

Comment:
Are we recording these durations in their own samples somewhere? If not, that could cause some issues when comparing client and server measured latencies. Right now any discrepancies can generally be attributed to network or client side issues, but that won't be true any longer.